[PR-12263][SEO] Meta description for docs pages (#190)
talkable/talkable-docs: source/web_hooks/claim_signup.rst

.. _web_hooks/claim_signup:
.. include:: /partials/common.rst

.. meta::

>    :description: The Talkable Friend Email Gating Webhook notifies your endpoint that friend email gating form was submitted.

Reviewer: ```suggestion
   :description: Friend Email Gating Webhook notifies your endpoint that a Friend email gating form was submitted.
```
